[2024-05-11 11:02:25] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:12:45] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:14:55] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:15:12] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:16:06] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:16:27] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:23:57] INFO [<module>: config.py, 57] 接口测试
[2024-05-11 11:24:55] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:25:43] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:29:42] INFO [<module>: config.py, 57] 接口测试
[2024-05-11 11:29:46] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:29:46] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:29:46] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:29:46] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:30:01] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"ad","password":"123456","remember":false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 402, 'message': '用户名或密码错误', 'response': None}
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 194
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"admin","password":"123456","remember":false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 1, 'message': '成功', 'response': {'id': None, 'userUuid': None, 'userName': 'admin', 'password': None, 'realName': None, 'age': None, 'sex': None, 'birthDay': None, 'userLevel': None, 'phone': None, 'role': None, 'status': None, 'imagePath': None, 'createTime': None, 'modifyTime': None, 'lastActiveTime': None, 'deleted': None, 'wxOpenId': None}}
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'student'
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 27] ((1, 'd2d29da2-dcb3-4013-b874-727626236f47', 'student', '61BD60C60D9FB60CC8FC7767669D40A1', '学生', 18, 1, datetime.datetime(2019, 9, 1, 16, 0), 1, '19171171610', 1, 1, 'https://www.mindskip.net:9008/image/ba607a75-83ba-4530-8e23-660b72dc4953/头像.jpg', datetime.datetime(2019, 9, 7, 18, 55, 2), datetime.datetime(2020, 2, 4, 8, 26, 54), None, b'\x00', None),)
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'student'
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 27] ((1, 'd2d29da2-dcb3-4013-b874-727626236f47', 'student', '61BD60C60D9FB60CC8FC7767669D40A1', '学生', 18, 1, datetime.datetime(2019, 9, 1, 16, 0), 1, '19171171610', 1, 1, 'https://www.mindskip.net:9008/image/ba607a75-83ba-4530-8e23-660b72dc4953/头像.jpg', datetime.datetime(2019, 9, 7, 18, 55, 2), datetime.datetime(2020, 2, 4, 8, 26, 54), None, b'\x00', None),)
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"student","password":"123456","userLevel":1}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 2, 'message': '用户已存在', 'response': None}
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 27] ()
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"peter","password":"123456","userLevel":1}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 1, 'message': '成功', 'response': None}
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 27] ((75, 'a4a23654-468e-4896-ad05-cd7a1b365498', 'peter', 'HPsbW5nZ4YhjRKg6AZruiG5oeyeaKBRYmqSMoYvj2LQCXciuIuv8fAsQ0FbVWSL9T7BXQmrpzD33ydedr62CwiSzosx02EiApbyoVjXM8RnaMoXKz+Uk87TH3w8hoFX5VbKJbpYiO2V5w/E6omUvwAELtwn5mmXklleSLmClUO0=', None, None, None, None, 1, None, 1, 1, None, datetime.datetime(2024, 5, 11, 11, 30, 1), None, datetime.datetime(2024, 5, 11, 11, 30, 1), b'\x00', None),)
[2024-05-11 11:30:01] DEBUG [change_db: db1.py, 37] delete from t_user where user_name='peter'
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 194
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：login_ok
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "admin", "password": "123456", "remember": false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":1,"message":"成功","response":{"id":null,"userUuid":null,"userName":"admin","password":null,"realName":null,"age":null,"sex":null,"birthDay":null,"userLevel":null,"phone":null,"role":null,"status":null,"imagePath":null,"createTime":null,"modifyTime":null,"lastActiveTime":null,"deleted":null,"wxOpenId":null}}
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：login_err01
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "ad", "password": "123456", "remember": false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：login_err02
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "", "password": "123456", "remember": false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：用户名或密码错误
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：login_err03
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "peter", "password": "", "remember": false}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：用户名或密码错误
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:01] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 14] 测试用例：test_reg_err
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "student", "password": "123456", "userLevel": 1}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:01] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:01] DEBUG [query_db: db1.py, 27] ()
[2024-05-11 11:30:01] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:02] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:02] INFO [log_case_info: case_log.py, 14] 测试用例：test_reg_ok
[2024-05-11 11:30:02] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:02] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "peter", "password": "123456", "userLevel": 1}
[2024-05-11 11:30:02] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:02] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:02] DEBUG [change_db: db1.py, 37] delete from t_user where user_name='peter'
[2024-05-11 11:30:02] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:02] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:02] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:02] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:02] ERROR [run: run_all.py, 41] 测试失败，失败用例已保存到文件：D:\2226\apitest26\last_fails.pickle
[2024-05-11 11:30:03] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:30:03] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:30:25] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:30:25] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:25] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"ad","password":"123456","remember":false}
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 402, 'message': '用户名或密码错误', 'response': None}
[2024-05-11 11:30:25] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:25] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 194
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"admin","password":"123456","remember":false}
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 1, 'message': '成功', 'response': {'id': None, 'userUuid': None, 'userName': 'admin', 'password': None, 'realName': None, 'age': None, 'sex': None, 'birthDay': None, 'userLevel': None, 'phone': None, 'role': None, 'status': None, 'imagePath': None, 'createTime': None, 'modifyTime': None, 'lastActiveTime': None, 'deleted': None, 'wxOpenId': None}}
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'student'
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 27] ((1, 'd2d29da2-dcb3-4013-b874-727626236f47', 'student', '61BD60C60D9FB60CC8FC7767669D40A1', '学生', 18, 1, datetime.datetime(2019, 9, 1, 16, 0), 1, '19171171610', 1, 1, 'https://www.mindskip.net:9008/image/ba607a75-83ba-4530-8e23-660b72dc4953/头像.jpg', datetime.datetime(2019, 9, 7, 18, 55, 2), datetime.datetime(2020, 2, 4, 8, 26, 54), None, b'\x00', None),)
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'student'
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 27] ((1, 'd2d29da2-dcb3-4013-b874-727626236f47', 'student', '61BD60C60D9FB60CC8FC7767669D40A1', '学生', 18, 1, datetime.datetime(2019, 9, 1, 16, 0), 1, '19171171610', 1, 1, 'https://www.mindskip.net:9008/image/ba607a75-83ba-4530-8e23-660b72dc4953/头像.jpg', datetime.datetime(2019, 9, 7, 18, 55, 2), datetime.datetime(2020, 2, 4, 8, 26, 54), None, b'\x00', None),)
[2024-05-11 11:30:25] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:25] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"student","password":"123456","userLevel":1}
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:25] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 2, 'message': '用户已存在', 'response': None}
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:25] DEBUG [query_db: db1.py, 27] ()
[2024-05-11 11:30:25] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"peter","password":"123456","userLevel":1}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 1, 'message': '成功', 'response': None}
[2024-05-11 11:30:26] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:26] DEBUG [query_db: db1.py, 27] ((78, '4c2e76fe-1022-475b-9c45-a1e7c5234114', 'peter', 'mHaLVZi+6qYEyFHv0YdMzOjCXsyAssWiATplkmReXuNxhiD2wox0rSTnPWFN+lgajSJx7yUPDkmkpF+O/tiFV6GhyoPxQFJg80v/wZ6qbD9DDPPb0467umDVyaBrom0ihQqM0VBhBy5De66Vstxw2bIA9N3Wjs2UuSCub7nOH8A=', None, None, None, None, 1, None, 1, 1, None, datetime.datetime(2024, 5, 11, 11, 30, 26), None, datetime.datetime(2024, 5, 11, 11, 30, 26), b'\x00', None),)
[2024-05-11 11:30:26] DEBUG [change_db: db1.py, 37] delete from t_user where user_name='peter'
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 194
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：login_ok
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "admin", "password": "123456", "remember": false}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":1,"message":"成功","response":{"id":null,"userUuid":null,"userName":"admin","password":null,"realName":null,"age":null,"sex":null,"birthDay":null,"userLevel":null,"phone":null,"role":null,"status":null,"imagePath":null,"createTime":null,"modifyTime":null,"lastActiveTime":null,"deleted":null,"wxOpenId":null}}
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：login_err01
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "ad", "password": "123456", "remember": false}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：login_err02
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "", "password": "123456", "remember": false}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：用户名或密码错误
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：login_err03
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "peter", "password": "", "remember": false}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：用户名或密码错误
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：test_reg_err
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "student", "password": "123456", "userLevel": 1}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":2,"message":"用户已存在","response":null}
[2024-05-11 11:30:26] DEBUG [query_db: db1.py, 24] select * from t_user where user_name = 'peter'
[2024-05-11 11:30:26] DEBUG [query_db: db1.py, 27] ()
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 14] 测试用例：test_reg_ok
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/student/user/register
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "peter", "password": "123456", "userLevel": 1}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 17] 期望结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:26] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":1,"message":"成功","response":null}
[2024-05-11 11:30:26] DEBUG [change_db: db1.py, 37] delete from t_user where user_name='peter'
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:26] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:30:26] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/student/user/register HTTP/1.1" 200 None
[2024-05-11 11:30:26] ERROR [run: run_all.py, 41] 测试失败，失败用例已保存到文件：D:\2226\apitest26\last_fails.pickle
[2024-05-11 11:30:27] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:30:27] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:31:04] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:31:04] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:31:05] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:31:05] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:33:05] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:33:05] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:33:06] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:33:06] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:34:35] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:34:35] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 194
[2024-05-11 11:34:35] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:34:35] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:34:35] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"admin","password":"123456","remember":false}
[2024-05-11 11:34:35] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:34:35] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 1, 'message': '成功', 'response': {'id': None, 'userUuid': None, 'userName': 'admin', 'password': None, 'realName': None, 'age': None, 'sex': None, 'birthDay': None, 'userLevel': None, 'phone': None, 'role': None, 'status': None, 'imagePath': None, 'createTime': None, 'modifyTime': None, 'lastActiveTime': None, 'deleted': None, 'wxOpenId': None}}
[2024-05-11 11:36:07] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:36:07] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:36:07] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"ad","password":"123456","remember":false}
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 402, 'message': '用户名或密码错误', 'response': None}
[2024-05-11 11:36:07] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:36:07] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:07] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:36:07] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 14] 测试用例：login_err01
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "ad", "password": "123456", "remember": false}
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:36:07] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:36:07] ERROR [run: run_all.py, 41] 测试失败，失败用例已保存到文件：D:\2226\apitest26\last_fails.pickle
[2024-05-11 11:36:07] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:36:07] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:36:21] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:36:21] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:36:21] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"ad","password":"123456","remember":false}
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 402, 'message': '用户名或密码错误', 'response': None}
[2024-05-11 11:36:21] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:36:21] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:21] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:36:21] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 14] 测试用例：login_err01
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "ad", "password": "123456", "remember": false}
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:36:21] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:36:21] ERROR [run: run_all.py, 41] 测试失败，失败用例已保存到文件：D:\2226\apitest26\last_fails.pickle
[2024-05-11 11:36:22] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:36:22] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:37:56] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:37:56] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:37:56] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 14] 测试用例：None
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName":"ad","password":"123456","remember":false}
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 18] 实际结果：{'code': 402, 'message': '用户名或密码错误', 'response': None}
[2024-05-11 11:37:56] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 192.168.55.1:8000
[2024-05-11 11:37:56] DEBUG [_make_request: connectionpool.py, 546] http://192.168.55.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:37:56] DEBUG [_new_conn: connectionpool.py, 244] Starting new HTTP connection (1): 127.0.0.1:8000
[2024-05-11 11:37:56] DEBUG [_make_request: connectionpool.py, 546] http://127.0.0.1:8000 "POST /api/user/login HTTP/1.1" 200 85
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 14] 测试用例：login_err01
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 15] url：http://127.0.0.1:8000/api/user/login
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 16] 请求参数：{"userName": "ad", "password": "123456", "remember": false}
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 17] 期望结果：成功
[2024-05-11 11:37:56] INFO [log_case_info: case_log.py, 18] 实际结果：{"code":402,"message":"用户名或密码错误","response":null}
[2024-05-11 11:37:56] ERROR [run: run_all.py, 41] 测试失败，失败用例已保存到文件：D:\2226\apitest26\last_fails.pickle
[2024-05-11 11:37:57] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:37:57] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:38:38] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:38:38] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:38:39] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:38:39] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:41:04] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:41:04] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:41:05] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:41:05] INFO [run: run_all.py, 47] ====测试结束====
[2024-05-11 11:46:35] INFO [run: run_all.py, 32] ====测试开始====
[2024-05-11 11:46:35] INFO [run: run_all.py, 43] 测试成功
[2024-05-11 11:46:35] INFO [send_email: send_email.py, 49] ====================发送邮件成功======================
[2024-05-11 11:46:36] INFO [run: run_all.py, 47] ====测试结束====
